gokrazy.go

```diff
@@ -117,10 +141,25 @@ func Boot(userBuildTimestamp string) error {
 	}
 
 	initRemoteSyslog()
+    if err := setupTLS(); err != nil {
```

Comment:
Looks like you didn’t run gofmt on your code? In general, I’d recommend to set up your editor to automatically run `goimports` on save: https://godoc.org/golang.org/x/tools/cmd/goimports

Reply:
Hm, I actually do this as pre-commit hook. I don't know what happened here.